docs/source/mayavi/installation.rst

```diff
@@ -142,6 +142,58 @@ After installing VTK Python, you can install mayavi using `pip`::
 .. _PyPA: https://packaging.python.org/en/latest/current/
 .. _PyPI: https://pypi.python.org/pypi
 
+.. _install-with-edm:
+
+Installing with `Enthought Deployment Manager(EDM)`
+.....................
```

Comment:
Use more dots for the underlining.  You will see a warning when building the docs or see here for example: 
https://travis-ci.org/enthought/mayavi/jobs/259137173